modules/log6x-6-1-4-rn.adoc

```diff
@@ -0,0 +1,36 @@
+// Module included in the following assemblies:
+//
+// * observability/logging/logging-6.1/log6x-release-notes-6.1.adoc
+
+:_mod-docs-content-type: REFERENCE
+[id="logging-release-notes-6-1-4_{context}"]
+= Logging 6.1.4 Release Notes
+
+This release includes link:https://access.redhat.com/errata/RHSA-2025:3131[{logging-uc} {for} Bug Fix Release 6.1.4].
+
+[id="logging-release-notes-6-1-4-bug-fixes_{context}"]
+== Bug Fixes
+
+* Before this update, Red Hat Managed Elasticsearch failed to receive logs if the index name did not follow the required patterns (app-, infra-, audit-), resulting in an `index_not_found_exception` error due to a restricted automatic index creation. With this update, improved documentation and explanations in `oc explain obsclf.spec.outputs.elasticsearch.index` clarify the index naming limitations, helping users configure log forwarding correctly.
+(link:https://issues.redhat.com/browse/LOG-6623[LOG-6623])
+
+* Before this update, an error in the Operator generating the Loki configuration sets the number of delete workers to zero when using `1x.pico` as the LokiStack size. With this update, the number of delete workers is set to ten.
```

Comment:
```suggestion
* Previously, when you used `1x.pico` as the LokiStack size, the number of delete workers was set to zero. This issue occurred because of an error in the Operator that generates the Loki configuration. With this update, the number of delete workers is set to ten.
```